frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -24,11 +34,52 @@ const navigateTo = (e: Event, url: string) => {
   e.preventDefault();
 };
 
+const resourceAttributes = (
+  model: K8sKind,
+  namespace: string,
+  verb: K8sVerb,
```

Comment:
You might want to just always use `create` for this helper instead of making it configurable. To make it general for other verbs, you'd need to take in an optional resource name as well. It's not exported, and you only use `create` below. I'd suggest

```ts
const createResourceAttributes = (model: K8sKind, namespace: string): AccessReviewResourceAttributes => ({
  group: model.apiGroup,
  resource: model.plural,
  namespace,
  'create',
});
```

Sorry, I should have made this comment earlier.

Reply:
Done.